en/.spelling

```diff
@@ -0,0 +1,46 @@
+>
+Choreo
+API
+APIs
+COVID-19
+cInlineImage-full 
+microservices
+OpenAPI
+lifecycle
+OAuth 2.0
+developer-centric
+timeframe
+https
+integrations
+cInlineImage-half
+choreo-connectors-list
+prebuilt
+ForEach
+md
+cInlineImage-bordered
+backend
+SaaS
+LeanIX
+SendGrid
+subusers
+Spotify
+Gmail
+OAuth
+hostname
+millicores
+MiB
+forecasted
+e.g.
+64-bit
+SLAs
+Twilio
+WhatsApp
+NetSuite
+sublist
+storages
+EventHub
+i.e.
+stateful
+multiline
+
```

Comment:
Shall we add the following too:
GitHub
API Developer Portal 
Google Calendar 
open source 
on-premises

Reply:
Resolving this based on the chat we had regarding this at the team meeting.